Lib/test/test_hmac.py

```diff
@@ -719,14 +748,32 @@ class OpenSSLRFCTestCase(OpenSSLAssertersMixin,
     """
 
 
+class BuiltinRFCTestCase(BuiltinAssertersMixin,
+                         WithNamedHashFunctions, RFCTestCaseMixin,
+                         unittest.TestCase):
+    """Built-in HACL* implementation of HMAC.
+
+    The underlying hash functions are also HACL*-based.
+    """
+
+    def assert_hmac_extra_cases(
+        self, key, msg, hexdigest, digestmod, hashname, digest_size, block_size
+    ):
+        # assert one-shot HMAC at the same time
+        with self.subTest(key=key, msg=msg, hashname=hashname):
+            func = eval(f'self.hmac.compute_{hashname}')
```

Comment:
```suggestion
            func = getattr(self.hmac, f'compute_{hashname}')
```

I wanted to check whether the `eval` worked (and it works) but I forgot to revert it :)